mantidproject/mantid
# Engg::Add "cropped" calibration in EnggDiffraction

Fixes #14591 

**This should be merged after the following pull request has been merged #14904**
- [x] _#14904 merged_

This branch will enable `Interface/Diffraction/Engineering Diffraction/Calibration/``Calibration Cropped` to be processed, vanadium run number under `Cropped Calibration` which will be reading from the vanadium field, & user are unable to modify, the `specIds` selected are passed to the `EnggCalibration`.  

**To Test**
- Code Check
- GUI: `Calibration Cropped Sample #:` updated as soon as new Ceria is changed under `Make New Calibration` section
- GUI: `Spectrum IDs:` only enabled if the `Bank Name:` is set as `Enable Spectrum-IDs`
- The files are still generated how they are generated for standard calibration
- Make sure additional parameter is being passed when `Cropped Calibration` process is carried out
